acceptance-tests.md

```diff
@@ -9,25 +9,32 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Request a new account**
     * _As a new user, I should be able to create an account for myself in the NERC._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. A prospective user follows the steps documented in [https://nerc-project.github.io/nerc-docs/get-started/create-a-user-portal-account/](https://nerc-project.github.io/nerc-docs/get-started/create-a-user-portal-account/) to create an account on the NERC.
+    * Acceptance tests:
+        1. For a given username of `computate` for example, [Click here to check that the given username is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~User&name=computate). Note the identities for the given user, specifically one that starts with `github:`, like `github:15114160`.
```

Comment:
before step 1, you have to check the MSS Account portal to see that the MGHPCC-SS Account has been created, and view the assigned username.  There can be a delay of days between when the account is submitted and when it is created, which varies depending on the type of account approval required by the NERC.  Suggest a particular test user accountname (requires functional email) here for clarity.